Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -295,6 +295,27 @@ Advanced Configuration
 This section covers a few advanced configuration modes for the kube-proxy replacement
 that go beyond the above Quick-Start guide and are entirely optional.
 
+Policy Enforcement
```

Comment:
I don't think that this section is specific to kube-proxy-free, is it? Won't Cilium operate the same with kube-proxy too? I'm wondering if there's somewhere more generic we could mention this, possibly just in the policy intro section..? :thinking:

Reply:
With the recent changes they won't differ, just that the KPR provides some tools to work around the limitation of the direct-routing mode (e.g., DSR). Maybe creating a new page called "Caveats" which could be linked from https://docs.cilium.io/en/v1.11/policy/?